Source/WebCore/page/text-extraction/TextExtraction.cpp

```diff
@@ -926,11 +926,31 @@ static std::optional<SimpleRange> searchForText(Node& node, const String& search
     return { WTFMove(foundRange) };
 }
 
-static void dispatchSimulatedClick(Page& page, IntPoint location, CompletionHandler<void(bool)>&& completion)
+static String invalidNodeIdentifierDescription(NodeIdentifier identifier)
+{
+    return makeString("Failed to resolve nodeIdentifier "_s, identifier.loggingString());
+}
+
+static String nullFrameDescription()
+{
+    return "Browsing context has been detached"_s;
+}
```

Comment:
Nit: this one and `interactedWithSelectElementDescription` can be constexpr

Reply:
oh, nvm, sorry, this constructor is not constexpr sadly

```
String(ASCIILiteral);
```

Reply:
Oh, I can still make it a `constexpr` and return an `ASCIILiteral`!